proto/evm/v1beta1/query.proto

```diff
@@ -67,3 +67,8 @@ message QueryDepositStateResponse {
   string log = 1;
   DepositStatus status = 2;
 }
+
+message QueryAddressesResponse {
+  repeated string addresses = 1
+      [ (gogoproto.nullable) = false, (gogoproto.customtype) = "Address" ];
```

Comment:
Can you check to make sure we display in the console as proper ethereum addresses as string?

Reply:
done